src/realtime_metrics/main.py

```diff
@@ -146,7 +153,19 @@ def run_vehicle_position_analysis():
 
 def mse_accuracy(stop_time_updates: list[tuple[TripUpdate, StopTimeUpdate]]) -> float | None:
     """
-    computes the accuracy of the given stop time updates using mean squared error
+    Computes the accuracy of the given stop time updates using mean squared error. 
+    Stop time updates with unknown actual arrival time are omitted.
+
+    Parameters
+    ----------
+    stop_time_updates : list[tuple[TripUpdate, StopTimeUpdate]
+        a list of StopTimeUpdates with the corresponding TripUpdate
```

Comment:
It is already documented that it's a list, by the type hint, so no need to tell it again
```suggestion
        pairs of StopTimeUpdates and the corresponding TripUpdate
```